get_started.sh

```diff
@@ -75,6 +75,12 @@ sudo apt-get install ros-kinetic-desktop-full
 sudo rosdep init
 rosdep update
 
+# Run rosinstall for other packages
+sudo mkdir /usr/share/ros
```

Comment:
I think this will fail (and kill this script) if the directory already exists. We should be using `sudo mkdir -p` instead.

Reply:
What was the reasoning behind this path? I'd think we'd want to closely emulate the normal ros install path structure (so something more like `/usr/share/ros/kinetic/` (though obviously this is a bit different, as normally ros would install stuff to `/opt/ros/kinetic/share`, and we're already in a "share" folder).

Reply:
The first reason I chose this was to differentiate between what was installed via rosinstall and apt. The second reason being that I was worried the setup scripts would conflict with each other since they are setup the same way. If I could figure out a way to have rosinstall be smart enough to merge the setup scripts, then we could change the path to match. But we can add a share folder there, no big difference, just didnt see us needing other folders under `opt/ros/kinetic/` to have to go one dir deeper.